lib/waterline/adapter/types.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  supportsType: function(type) {
+    var adapter = this.connections[connName]._adapter;
+    return adapter.supportsType && adapter.supportsType(type);
```

Comment:
So, won't this return `false` for adapters which haven't yet implemented this new method?